bin/single-new-eks-opensource-observability.ts

```diff
@@ -1,4 +1,4 @@
-import SingleNewEksOpenSourceobservabilityConstruct from '../lib/single-new-eks-opensource-observability-construct';
+import SingleNewEksOpenSourceobservabilityConstruct from '../lib/single-new-eks-opensource-observability';
```

Comment:
Why is class name `SingleNewEksOpenSourceobservabilityConstruct ` still having Construct ? We want to totally get rid of word Construct with Pattern. Also we dont want -construct to be removed. We would rather want it to be renamed to -pattern.